source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.ApplyDBMigrationsApp/GreenEnergyHub.Charges.ApplyDBMigrationsApp.csproj

```
@@ -48,6 +48,7 @@ limitations under the License.
       <EmbeddedResource Include="Scripts\Model\202206301048 Migrate AvailableData ActorId to reference MarketParticipant ActorId.sql" />
       <EmbeddedResource Include="Scripts\Model\202207051352 Remove Foreign Keys on DataAvailable tables.sql" />
       <EmbeddedResource Include="Scripts\Model\202207271035 Remove position from ChargePoint.sql" />
+      <EmbeddedResource Include="Scripts\Model\202208101116 Create outbox table.sql" />
```

```suggestion
      <EmbeddedResource Include="Scripts\Model\202208101116 Create OutboxMessage table.sql" />
```
Remember to rename the file, too.